inst/private/python_ipc_pytave.m

```diff
@@ -70,7 +70,6 @@
 
   s = strjoin(cmd, newl);
   pyexec(s)
-  
-  info.raw = pyeval('str(_outs)');
```

Comment:
If we are removing info. We should remove `info.prelines` too (you probably missed it)